.github/azure-gpu-tests.yml

```
@@ -50,7 +50,7 @@ jobs:
         python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu == 2, f'GPU: {mgpu}'"
       displayName: 'Image info & NVIDIA'
 
-    - script: pip install pytest -r requirements.txt
+    - script: pip install .[all] pytest
```

Does this need to be `pip install ".[all]" pytest` (I guess it depends on the shell)

---

Linux not but Mac yes... anyway we can have it :)
```suggestion
    - script: pip install ".[all]" "pytest"
```